### ISSUE TITLE
Warp stops behaving as expected loosing its differential features

### ISSUE THREAD

**Issue by davidcalap** [warpdotdev/Warp#2736]
### Discord username (optional)

palacd#0703

### Describe the bug

Warp loses its normal behavior (separated blocks for command outputs and so on) and turns into a classic terminal with just one block and look & feel after running aws-vault exec command and executing k9s (https://github.com/derailed/k9s) a terminal-ui GO application.

### To Reproduce

Open warp
Run aws-vault exec command to login into AWS
Run k9s in warp
Exit k9s

### Expected behaviour

Warp still has the same appearance and blocks behavior 

### Screenshots

<img width="1868" alt="image" src="https://user-images.githubusercontent.com/124673177/222687106-42a50ad1-a06d-4d51-8e96-dbb923bd8ed5.png">

<img width="1868" alt="image" src="https://user-images.githubusercontent.com/124673177/222687188-9f7e8d9e-a7c6-4beb-889d-22cafa850559.png">

<img width="1868" alt="image" src="https://user-images.githubusercontent.com/124673177/222687453-88164891-f9b1-43a1-8ec0-938507aaca14.png">

### Operating System

MacOS

### OS Version

13.2.1

### Warp Version

v0.2023.02.28.08.03.stable_01

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
We appreciate you submitting this @davidcalap 
This is related to the lack of subshell support and we're tracking this issue here #64 

TLDR, It's a bit tricky but we're working on supporting Warp features within subshells, so you should see some improvements coming in the not-too-distant future.

Please :+1: and comment there if you're also seeing the issue, as it helps us gauge the impact of the issue.

Closing as duplicate.
